llvm/lib/Analysis/Lint.cpp

```diff
@@ -139,7 +139,7 @@ class Lint : public InstVisitor<Lint> {
 
   Lint(Module *Mod, const DataLayout *DL, AliasAnalysis *AA,
        AssumptionCache *AC, DominatorTree *DT, TargetLibraryInfo *TLI)
-      : Mod(Mod), TT(Triple::normalize(Mod->getTargetTriple())), DL(DL), AA(AA),
+      : Mod(Mod), TT(Mod->getTargetTriple().normalize()), DL(DL), AA(AA),
```

Comment:
I don't understand the call to `normalize` here. `Triple->std::string->Triple` looks unnecessary.